src/Filesystem/DTO/FileSize.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace ILIAS\Filesystem\DTO;
```

Comment:
@chfsx: The name was new to me and we also seem to be in another usecase (at least according to [wikipedia](https://en.wikipedia.org/wiki/Data_transfer_object).